1-exercises/A-reading-list/script.js

```diff
@@ -1,5 +1,37 @@
+
 function readingList(books) {
-  // Write your code here...
+  const bookDiv = document.querySelector("#content");
+  const listOfBooks = document.createElement('ul');
+  bookDiv.appendChild(listOfBooks);
+  for (let book of books){
+    let bookTitle = document.createElement("p");
+    bookTitle.innerText = `${book.title} by ${book.author}`;
+
+    let bookImg = document.createElement("img");
+    bookImg.src = book.bookCoverImage;
+
+    let liElement = document.createElement("li");
+    liElement.appendChild(bookTitle);
+    liElement.appendChild(bookImg);
+
+    listOfBooks.appendChild(liElement);
```

Comment:
Comment: Interesting. I see you put the list of li before the if ... else statement. Well done!

Consider: Another way to do is:

if (book.alreadyRead) {
      item.style.backgroundColor = 'green';
    } else {
      item.style.backgroundColor = 'red';
    }

    list.appendChild(item);
  });

  content.appendChild(list);
}

https://github.com/CodeYourFuture/JavaScript-Core-2-Coursework-Week3/pull/215/files